src/battle/partner/watt.c

```diff
@@ -11,6 +11,7 @@ extern s32 D_8023C1C8;
 extern EffectInstance* D_8023C1CC;
 extern EffectInstance* D_8023C1D0;
 extern EffectInstance* D_80239A0C_7054FC;
+extern EffectInstance* D_8023C1B0;
```

Comment:
Could you move this to be the first extern in the list (so it's in memory order)? B0, B4, etc